cmd/busyambassador/main.go

```diff
@@ -13,11 +13,11 @@ import (
 	"strings"
 
 	// 1st-party libs
+	"github.com/datawire/ambassador-agent/cmd/agent"
```

Comment:
that seems reasonable, I don't know enough to make that decision. If you want me to remove it, I can

Reply:
Yeah, let's remove this.

Reply:
Yeah, I would say that the only dependency that emissary should have on the agent going forward should be the container tags (_at least based on current design_)

The public contract between the two consists of a couple pulls (snapshots, diagnostics) and push (envoy stats) so we should probably connect offline and get those documented so that we make sure those stay aligned.